docs/Parameters-Tuning.rst

```diff
@@ -36,15 +36,145 @@ To get good results using a leaf-wise tree, these are some important parameters:
 For Faster Speed
 ----------------
 
--  Use bagging by setting ``bagging_fraction`` and ``bagging_freq``
+Add More Computational Resources
+''''''''''''''''''''''''''''''''
 
--  Use feature sub-sampling by setting ``feature_fraction``
+On systems where it is available, LightGBM uses OpenMP to parallelize many operations. The maximum number of threads used by LightGBM is controlled by the parameter ``num_threads``. By default, this will defer to the default behavior of OpenMP (one thread per real CPU core or the value in environment variable ``OMP_NUM_THREADS``, if it is set). For best performance, set this to the number of real CPU cores available.
```

Comment:
```suggestion
On systems where it is available, LightGBM uses OpenMP to parallelize many operations. The maximum number of threads used by LightGBM is controlled by the parameter ``num_threads``. By default, this will defer to the default behavior of OpenMP (one thread per real CPU core or the value in environment variable ``OMP_NUM_THREADS``, if it is set). For best performance, set this to the number of **real** CPU cores available.
```